evm/v0.5/package.json

```diff
@@ -8,6 +8,7 @@
   },
   "devDependencies": {
     "chai": "^4.2.0",
+    "openzeppelin-solidity": "^2.3.0",
```

Comment:
Which will let us get rid of this dependency.

Reply:
D'oh. Thanks, I'll fix it up.

Reply:
This had something to do with my node package state. I was still getting a solidity 0.4.24 `openzeppelin` dependency from the test-helpers package after fixing the `LinkEX` source, and also that apparent babel failure I mentioned on slack. Reinstalling chainlink from scratch seems to have fixed both issues.

I'll keep the broken install around for a while, in case we want to diagnose the failure.